app/assets/javascripts/application.js

```diff
@@ -11,6 +11,7 @@
 // about supported directives.
 //
 //= require jquery
+//= require bootstrap-sprockets
```

Comment:
Just saying because this hit me when building the RPM prepared for containerizing Portus 😉